File: people-among-dragons/src/models/publication-comment.ts
export class PublicationComment {
    _commentId: number;
    _commentText: string;
    _userName: string;
    _userEmail: string;
    _creationDate: Date;

    constructor(commentId: number, commentText: string, userName: string, userEmail: string, creationDate: Date){
        this._commentId = commentId;
        this._commentText = commentText;
        this._userName = userName;
        this._userEmail = userEmail;
        this._creationDate = creationDate;
    }
}

File: Dragon-WebApi/Models/PublicationComment.cs
namespace Dragon_WebApi.Models
{
    public class PublicationComment
    {
        public int Id { get; set; }
        public string CommentText { get; set; }
        public string UserName { get; set; }
        public string UserEmail { get; set; }
        public DateTime CreationDate { get; set; }

        public PublicationComment()
        {
            CommentText = "";
            UserName = "";
            UserEmail = "";
            CreationDate = DateTime.Now;
        }
    }
}
